avaframe/runScripts/runAnalyzeSimilaritySol.py

```diff
@@ -0,0 +1,82 @@
+"""
+    Compare the com1DFA kernel results to the similarity solution
+    This script computes the similarity solution for a gliding avalanche on
+    a inclined plane according to similarity solution from :
+    Hutter, K., Siegel, M., Savage, S.B. et al.
+    Two-dimensional spreading of a granular avalanche down an inclined plane
+    Part I. theory. Acta Mechanica 100, 37–68 (1993).
+    https://doi.org/10.1007/BF01176861
+    and compares it to the DFA kernel com1DFA
+"""
+
+import pathlib
+import pandas as pd
+
+# Local imports
+import avaframe.in3Utils.initializeProject as initProj
+import avaframe.in3Utils.fileHandlerUtils as fU
+from avaframe.in3Utils import cfgUtils
+from avaframe.in3Utils import logUtils
+import avaframe.com1DFA.com1DFA as com1DFA
+from avaframe.in1Data import getInput as gI
+import avaframe.ana1Tests.simiSolTest as simiSolTest
+import avaframe.out3Plot.outAna1Plots as outAna1Plots
+
+
+# +++++++++SETUP CONFIGURATION++++++++++++++++++++++++
+# log file name; leave empty to use default runLog.log
+logName = 'runSimilarityTest'
+
+# Load general configuration
+cfgMain = cfgUtils.getGeneralConfig()
+avalancheDir = 'data/avaSimilaritySol'
+
+# Clean input directory(ies) of old work and output files
+initProj.cleanSingleAvaDir(avalancheDir, keep=logName, deleteOutput=False)
+
+# Start logging
+log = logUtils.initiateLogger(avalancheDir, logName)
+log.info('MAIN SCRIPT')
+log.info('Current avalanche: %s', avalancheDir)
+
+# Load configuration for similarity solution test
+simiSolCfg = pathlib.Path(avalancheDir, 'Inputs', 'simiSol_com1DFACfg.ini')
+
+# create output directory for test result plots
+outDirTest = pathlib.Path(avalancheDir, 'Outputs', 'ana1Tests')
+fU.makeADir(outDirTest)
+
+cfg = cfgUtils.getModuleConfig(com1DFA, simiSolCfg)
+
+# Define release thickness distribution
+demFile = gI.getDEMPath(avalancheDir)
+simDF, _ = cfgUtils.readAllConfigurationInfo(avalancheDir)
+
+# compute the similartiy solution (this corresponds to our reference)
+log.info('Computing similarity solution')
+solSimi = simiSolTest.mainSimilaritySol(simiSolCfg)
+
+# if the analysis already exists and you only want to replot uncoment this
+# pathToResults = pathlib.Path(avalancheDir, 'Outputs', 'ana1Tests', 'results10.p')
```

Comment:
is it always for 10s? or should this name be build or read?

Reply:
No this can change. You can decide to analyze the results for other times